specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/extensionTypes/readme.python.md

```diff
@@ -0,0 +1,24 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+namespace: azure.mgmt.kubernetesconfiguration.extensiontypes
+package-name: azure-mgmt-kubernetesconfiguration-extensiontypes
+no-namespace-folders: true
+package-version: 1.1.0
+clear-output-folder: true
+```
+
+### Tag: package-preview-2024-11-01 and python
+
+These settings apply only when `--tag=package-preview-2024-11-01 --python` is specified on the command line.
+
```

Comment:
"azure-mgmt-kubernetesconfiguration" is multiapi package for python. If we split it into several independent packages, CLI may be influenced.
Add @kairu-ms for awareness.

Reply:
@dipti-pai Yeah. Once the PR merged, Python SDK team will only release for new package in the future. Since your CLI extension uses source code of package ‘azure-mgmt-kubernetesconfiguration’, you may need some adaption work in advance before developing new features.